internal/providers/terraform/azure/testdata/frontdoor_test/frontdoor_test.tf

```diff
@@ -11,7 +11,6 @@ resource "azurerm_resource_group" "example" {
 resource "azurerm_frontdoor" "frontdoor_without_usage" {
   name                                         = "exampleFrontdoor"
   resource_group_name                          = azurerm_resource_group.example.name
-  enforce_backend_pools_certificate_name_check = false
```

Comment:
Deprecated and removed: https://registry.terraform.io/providers/hashicorp/azurerm/latest/docs/guides/3.0-upgrade-guide#resource-azurerm_frontdoor.

Not used in the resource, decided to just remove.